chpl/chpl-service/src/main/java/gov/healthit/chpl/questionableactivity/ListingQuestionableActivityProvider.java

```diff
@@ -468,6 +468,24 @@ public QuestionableActivityListingDTO checkRealWorldTestingResultsRemoved(
         return activity;
     }
 
+    public QuestionableActivityListingDTO checkRealWorldTestingPlanAddedForNotEligibleListing(CertifiedProductSearchDetails origListing, CertifiedProductSearchDetails newListing) {
+        QuestionableActivityListingDTO activity = null;
+        if (StringUtils.isEmpty(origListing.getRwtPlansUrl()) && !StringUtils.isEmpty(newListing.getRwtPlansUrl())) {
+            activity = new QuestionableActivityListingDTO();
+            activity.setAfter("Added Real World Testing Plans URL for Listing not Real World Testing Eligible: " + origListing.getRwtPlansUrl());
```

Comment:
I may be just not following the code exactly, but did you want the activity.setAfter() string to have the newListing.getRwtPlansUrl() instead of the one from the origListing (which I assume is an empty string if it's being added)? Same question for line 484 below.